### PR TITLE
edits doc - model overview + figure

### PR DESCRIPTION
**Pull request by yairchn**

### CODE REVIEW

File: docs/src/Overview.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * Non-hydrostatic (i.e. unbalanced by gravity) pressure effects
+ * Entrainment and detrainment - mass exchange between the subdomains
+ * Eddy diffusivity and mixing length to account for turbulent mixing
+ * Microphysical cloud process: formation and removal of precipitation
+ * Radiative effect
+
+It is these closures that are the foci of our machine learning effort, and specifically the entrainment and detrainment that are notorious effecting the global behavior of climate models.
+
+A conceptual sketch of the model in a climate model grid box is shown in the figure below, with the projected distribution of the combined environment and updrafts. Also sketched are the various physical processes requiring closures.
+![alt text](https://github.com/CliMA/TurbulenceConvection.jl/blob/yc/docs_edit/combined_globe_edmf.jpg?raw=true)
```
Comment (charleskawczynski) [inline docs/src/Overview.md:18]:
```suggestion
![alt text](figures/combined_globe_edmf.jpg)
```

File: docs/src/Formulation.md [hunk 2]
```diff
@@ -1,3 +1,1130 @@
-# TurbulenceConvection.jl mathematical formulation
+```math
```
Comment (charleskawczynski) [inline docs/src/Formulation.md:1]: Can we use `git mv docs/src/EDMFEquations.md docs/src/Formulation.md` to reduce the diff?

File: docs/src/Overview.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# TurbulenceConvection.jl
```
Comment (charleskawczynski) [inline docs/src/Overview.md:1]: Need to add `Overview.md` to the `pages` in `docs/make.jl`

File: docs/src/Overview.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+```@meta
+CurrentModule = EDMF Overview
+```
```
Comment (charleskawczynski) [inline docs/src/Overview.md:5]:
```suggestion
```

File: docs/src/Overview.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+ * Entrainment and detrainment - mass exchange between the subdomains
+ * Non-hydrostatic (i.e. unbalanced by gravity) pressure effects
+ * Eddy diffusivity and mixing length to account for turbulent mixing
+ * Microphysical cloud process such as the formation and removal of precipitation
+ * Radiative effects
+
+These closures that are the foci of our machine learning effort, and specifically the entrainment and detrainment that are notorious effecting the global behavior of climate models.
+
+A conceptual sketch of the model in a climate model grid box is shown in the figure below, with the projected distribution of the combined environment and updrafts. Also sketched are the various physical processes requiring closures.
+![alt text](figures/combined_globe_edmf.pdf)
```
Comment (charleskawczynski) [inline docs/src/Overview.md:15]: Please delete the `.eps` if we're going to use the pdf. Also, `alt text` I think is supposed to be a description of the image?

**Review by charleskawczynski — APPROVED** [review]
LGTM once we delete the duplicate image

**Comment by yairchn** [comment]
Bors r-

**Pull request merged**